lib/api.db.mysql.js

```diff
@@ -140,10 +140,10 @@ if (api.mysql) {
               }
             }
             fieldDef = fieldType.type;
-            if ('size' in field) fieldDef += '(' + field.size + ')';
+            if (field.size) fieldDef += '(' + field.size + ')';
             fieldDef += ' ';
-            if (('signed' in field) && !field.signed) fieldDef += 'unsigned ';
-            if ((('nullable' in field) && !field.nullable) || field.master) {
+            if (field.signed && !field.signed) fieldDef += 'unsigned ';
```

Comment:
That is a really strange condition, does it ever evaluate to true?
And even if it does, I think you should change it to be more clear about your intentions.